06-building-a-program-start-to-finish/setting-up-linters/output/README.md

```diff
@@ -0,0 +1,32 @@
+#### Output
+
+**Using**: Terminal & GitHub
+
+**Deliver**: Two screenshots of ESLint Airbnb style violations in the terminal and HoundCI comment on a PR
+
+**With**: *independently*
+
+
+####Link to GitHub Repo
+- [ andela-uibrahim / invertedIndex](https://github.com/andela-uibrahim/inverted-index/tree/chore/1/setting-up-repository)
+
+
+**screenshots**
+
+##Eslint Airbnb violation on the terminal
+
+- [andela-uibrahim/ Eslint Airbnb violation](https://photos.google.com/photo/AF1QipPurnaIaTiEZtKgqoVd1dP8q7qZfiM7WtXE8G4)
+<img src="https://lh3.googleusercontent.com/JYU70hzEjEtvi4TPVFETUp6q7vGCwdMcEQzSJTljLicAjdjWU4VbbDSRqqz6OJ8u24FaTqX1iNfqwK554iW26hCTjjNrkBSdRLRRqe5BqqHwcywrsp8-yj4H2SY37sovQQl7e_5KQ2Idn__yXR32zZUVVqtMsgG6IEafmy-RCpSoby0NHzLmmTyA38Kqyws6mjVfIHyLcq1WHYeFJvQsCvT-Q0KE8ND0OdoFPcfYV-vgWWordpy9Y_zNj8E8qFKO056XwkuYhuxR1zl6S06RCDgPQTS2BPrRtqb_AmWACA3T3LaVZqSKJMoAaRAH7LgCs8VYQjClGVpfzzRQOvpFSvYWMXgc7cwPWIOSqU3TTECTf2UBr2p7mEl2WzDe7SfKYsdHUgLBi6g230c6N4be2gsVpj79C8CVJEEwTAR5FWNsUx_sLI6LiV1vReGgPe2IqSSyyZFN8Wst2vW0mS7UMKBLmC_pKRcfPBnoMviojwdEg1bV3DLWDvfSfftw1xqhFH-M6_jtVhWZb-_DN_lXjpDUr-oqeBc7rxu6FzjDM0rMKClDTQNUztstHpDd8D_idfI6NnErcixDqwbJ0UY0w2X0X5ChzF2P06FUS4KUzjFk=w705-h264-no"/>
+
+
+
+
+##HoundCL comment on PR
+
+- [andela-uibrahim/ HoundCL](https://photos.google.com/photo/AF1QipO2HMwbEauOVygu7Ebitu8VCAwObsaljeqv-d4)
+<img src="https://lh3.googleusercontent.com/IWNUEcibgcKo1vA-2XiSNmDbD1UM5cHxuxuWk-RK-RphPkibbM7KQelCFZh9MHQh1W0OfeKSI57vblGPE3dZiaHUX5rY5JhGFNUZ7EpxNz71g5DkHqeOu-US9Axkie2jZ8uK0hhic3lunkxBlJVvx_Qga80MZ83Ru3yg15LF9aQSTDWMvF2ay3Y-stiN_R0rkb7hUuY9Pc4s0t3XT49z8Wnern4XhmOK9xZMJqAOI_sAMVCYgDC85fWuB-7UgH-Iy8HCXuHj8O87l2Bgbudnleu5ZWF8ZN3F_A26j0qmD_KxMJOtmU0U0UDiurvrwlVO_eFSgCOyUYjVLuNAKtysKTQEp_RkzRVPNyeRUH_8Z9dOXkv1E8g9x5o_0R5SR1sHXcyg2KYgFp99ioAZJ9aTJ7moiK6zyAa_kIrr10EW1tpgLqZtmAAaeNaFK3ordZesrf7tuo8bUP6iaX950987M6D_0g3TH5m5bTySxekJCSbRsslG0v0BYfnyX19WXG5__xM--3NvS8oywS1oFK01_rR4nmgSawglW-HKm_p9J9G1W9BQ6i56RHfliFVyj2yNuI3sEtUKdSVbZvzQfqcry4wN2SS2Uv8Kb_0izdA27OWF=w964-h659-no" />
```

Comment:
@q-ode I was trying to upload the images to view on my read me file. but I have rectified that now